src/components/layout/BaseLayout.tsx

```diff
@@ -1,22 +1,55 @@
 import React, { FC, PropsWithChildren, ReactNode } from 'react';
 
+import { chakra, Container, Flex } from '@chakra-ui/react';
+
 import Divider from '../divider';
+import Box from '../box';
+import { sizes } from '../../themes/shared/sizes';
 
 interface Props {
   header: ReactNode;
   footer?: ReactNode;
+  skyScraperAds?: ReactNode;
```

Comment:
How about:
```suggestion
  skyScraperAd?: ReactNode;
```
Or could there be more ads?